### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by open-schnick**
Dont merge just comment on it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,83 @@
...
+```shell script
+sudo snap install docker
+```
+##### Install with Apt
+Installing with [apt](https://en.wikipedia.org/wiki/APT_(software)) is a bit more difficult you can read [here](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-Docker-on-ubuntu-20-04) more about it.
 
 ## Usage
+After successfully installing Docker you can start using FileFighter.  
+This Repository contains one <!-- three --> important script:
+* The [initial start](./init_setup.sh) script starts downloads all the application and starts them in different containers. Run it with
```
Comment (qvalentin) [inline README.md:45]: ...starts downloading all the applications

File: README.md [hunk 2]
```diff
@@ -2,8 +2,83 @@
...
+After starting the script you should see something like the following:
+```shell script
+-------------------------< FileFighter >--------------------------
+|              Version 1.0 Last updated at 14.10.20              |
+|         Developed by Gimleux, Valentin, Open-Schnick.          |
+|       Development Blog: https://filefighter.github.io          |
+|  The code can be found at: https://www.github.com/filefighter  |
+--------------------< Started Initial Setup >---------------------
+```
+After the script succeeds you should be able to see the FileFighter Application.
```
Comment (qvalentin) [inline README.md:58]: Where?

File: README.md [hunk 3]
```diff
@@ -2,8 +2,83 @@
...
+
```
Comment (qvalentin) [inline README.md:59]: What do you need to do after installation? Login as admin and create users.

**Review by qvalentin — APPROVED** [review]
See comments

File: README.md (hunk 3 above)
Comment (open-schnick) [reply README.md:59]: > Login as admin and create users.

The question is how can we do that? Are we setting a default user as admin?

Comment (Gimleux) [reply README.md:62]: Maybe we could use Nextcloud's workflow for orientation. You create an admin user while setup who can edit the settings as well as add more users to the cloud manually

Comment (open-schnick) [reply README.md:62]: And how are we going to implement that?

Comment (Gimleux) [reply README.md:62]: Deploying the nas including a default admin user admin:admin; password needs to be changed on first login?

Comment (open-schnick) [reply README.md:62]: so a flag in the db for each use. Something like:
`"needsToChangePw": true`

Comment (Gimleux) [reply README.md:62]: for example. Another option would be add some lines in the password validation process which check if the password is still the default password

Comment (open-schnick) [reply README.md:62]: So we just set one user as default user and check it in the frontend if its the default one.

File: README.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ Installing with [apt](https://en.wikipedia.org/wiki/APT_(software)) is a bit mor
 ## Usage
 After successfully installing Docker you can start using FileFighter.  
 This Repository contains one <!-- three --> important script:
-* The [initial start](./init_setup.sh) script starts downloads all the application and starts them in different containers. Run it with 
+The [initial start](./init_setup.sh) script starts downloading all the application and starts them in different containers. Run it with
```
Comment (qvalentin) [inline README.md:46]: applicationS

File: README.md [hunk 5]
```diff
@@ -55,7 +55,10 @@ After starting the script you should see something like the following:
 |  The code can be found at: https://www.github.com/filefighter  |
 --------------------< Started Initial Setup >---------------------
 ```
-After the script succeeds you should be able to see the FileFighter Application.
+After the script succeeds you should be able to see the FileFighter Application in your browser.
```
Comment (qvalentin) [inline README.md:59]: application small

**Pull request merged**